### ISSUE TITLE
Add getNumberTypeCustom to TypeScript export

### ISSUE THREAD

**Issue by danielsefton** [catamphetamine/libphonenumber-js#153]
I noticed that getNumberTypeCustom is missing in "index.d.ts". I added the following and it works:

`export function getNumberTypeCustom(phone: TelephoneNumber, country: CountryCode, metadata: object): string;`

I'm not sure if there are other functions that need adding as well, possibly getPhoneCodeCustom.

Thanks!

**Comment by catamphetamine** [comment]
Yeah, as it has been stated, typescript is only supported via PRs
https://github.com/catamphetamine/libphonenumber-js#typescript
